tcms/testruns/static/testruns/js/get.js

```diff
@@ -340,32 +367,37 @@ function getExpandArea(testExecution) {
             })
         })
 
-    const textArea = container.find('textarea')[0]
-    const fileUpload = container.find('input[type="file"]')
-    const editor = initSimpleMDE(textArea, $(fileUpload), textArea.id)
     const commentsRow = container.find('.comments')
+    const simpleMDEinitialized = container.find('.comment-form').data('simple-mde-initialized')
+    if (!simpleMDEinitialized) {
+        const textArea = container.find('textarea')[0]
+        const fileUpload = container.find('input[type="file"]')
+        const editor = initSimpleMDE(textArea, $(fileUpload), textArea.id)
+        container.find('.comment-form').data('simple-mde-initialized', true)
+
+        container.find('.post-comment').click(() => {
+            const input = editor.value().trim()
+
+            if (input) {
+                jsonRPC('TestExecution.add_comment', [testExecution.id, input], comment => {
+                    editor.value('')
+
+                    commentsRow.append(renderCommentHTML(
+                        1 + template.find('.js-comment-container').length,
```

Comment:
Posting a comment after a fresh page load fails with `template is not defined`.

Next time test this before pushing.